### PR TITLE
LPD-31736 - Make regex lazy so that it doesn't replace content between two language imports

### PR DESCRIPTION
**Pull request by bryceosterhaus**

### CODE REVIEW

**Comment by bryceosterhaus** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/language/LanguageImpl.java [hunk 1]
```diff
@@ -2017,7 +2017,7 @@ private void _updateLastModified() {
 		LanguageImpl.class.getName() + "._groupLocalesPortalCache";
 
 	private static final String _LIFERAY_LANGUAGE_IMPORT_REGEXP =
-		"await import\\(.@liferay/language/.+/all\\.js.\\)";
+		"await import\\(.@liferay/language/.+?/all\\.js.\\)";
```
Comment (izaera) [inline portal-impl/src/com/liferay/portal/language/LanguageImpl.java:2020]: Just for the record: I've double tested the regexp at https://www.freeformatter.com/java-regex-tester.html and it seems correct. :muscle:

**Review by izaera — APPROVED** [review]

**Comment by izaera** [comment]
ci:report:40317073

**Comment by izaera** [comment]
Only one unique failure -> `com.liferay.portal.security.auto.login.test.RememberMeAutoLoginTest`
Since it could be related I'll investigate it.

**Comment by izaera** [comment]
The failing test passed locally and it's not related to JavaScript because it's testing backend objects using an integration test, so this PR is safe.

**Comment by izaera** [comment]
ci:forward:force

**Pull request closed without merging**